ui/components/multichain/notification-list-item/notification-list-item.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+
+import { Box, Icon, IconName, Text } from '../../component-library';
+import {
+  AlignItems,
+  BlockSize,
+  Display,
+  FlexDirection,
+  FontWeight,
+  JustifyContent,
+  IconColor,
+  TextColor,
+  TextVariant,
+  TextAlign,
+} from '../../../helpers/constants/design-system';
+import type { NotificationListItemIconProps } from '../notification-list-item-icon/notification-list-item-icon';
+import type { NotificationListItemTextProps } from '../notification-list-item-text/notification-list-item-text';
+import { NotificationListItemIcon } from '../notification-list-item-icon';
+import { NotificationListItemText } from '../notification-list-item-text';
+import { formatMenuItemDate } from '../../../helpers/utils/notification.util';
+
+export interface NotificationListItemProps {
+  id: string;
+  isRead: boolean;
+  icon: NotificationListItemIconProps;
+  title: NotificationListItemTextProps;
+  description: NotificationListItemTextProps;
+  createdAt: Date;
+  amount?: string;
+}
+
+/**
+ * `NotificationListItem` is a component that displays a single notification item.
+ *
+ * @param props - The properties object.
+ * @param props.isRead - Indicates whether the notification has been read.
+ * @param props.icon - The icon of the notification.
+ * @param props.title - The title of the notification.
+ * @param props.description - The description of the notification.
+ * @param props.createdAt - The date of the notification.
+ * @param props.amount - The amount associated with the notification, if applicable.
+ * @param props.id - The id of the notification.
+ * @returns Returns a notification list item component.
+ */
+export const NotificationListItem = ({
+  id,
+  isRead,
+  icon,
+  title,
+  description,
+  createdAt,
+  amount,
+}: NotificationListItemProps) => {
+  const handleClick = () => {
+    console.log('clicked', id);
```

Comment:
As a cleanup, lets remove all console.logs once we start integrating these components.

Note to others - these components are not connected in any way inside the extension. But will soon be integrated in followup PRs.